spec/views/rb_taskboards/show_spec.rb

```diff
@@ -137,14 +145,13 @@
 
       stories.each do |story|
         expect(rendered).to have_selector "#story_#{story.id}" do
-          with_selector ".assigned_to_id", assignee.name
+          with_selector '.assigned_to_id', assignee.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.